core/src/main/java/com/sequenceiq/cloudbreak/structuredevent/job/StructuredSynchronizerJob.java

```diff
@@ -0,0 +1,76 @@
+package com.sequenceiq.cloudbreak.structuredevent.job;
+
+import javax.inject.Inject;
+
+import org.quartz.DisallowConcurrentExecution;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.quartz.TracedQuartzJob;
+import com.sequenceiq.cloudbreak.service.stack.StackService;
+import com.sequenceiq.cloudbreak.structuredevent.LegacyDefaultStructuredEventClient;
+import com.sequenceiq.cloudbreak.structuredevent.StructuredSyncEventFactory;
+import com.sequenceiq.cloudbreak.structuredevent.event.StructuredSyncEvent;
+import com.sequenceiq.flow.core.FlowLogService;
+
+import io.opentracing.Tracer;
+
+@DisallowConcurrentExecution
+@Component
+public class StructuredSynchronizerJob extends TracedQuartzJob {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StructuredSynchronizerJob.class);
+
+    private String localId;
+
+    @Inject
+    private StackService stackService;
+
+    @Inject
+    private FlowLogService flowLogService;
+
+    @Inject
+    private StructuredSyncEventFactory structuredSyncEventFactory;
+
+    @Inject
+    private LegacyDefaultStructuredEventClient legacyDefaultStructuredEventClient;
+
+    public StructuredSynchronizerJob(Tracer tracer) {
+        super(tracer, "Structured Synchronizer Job");
+    }
+
+    @Override
+    protected Object getMdcContextObject() {
+        return stackService.getById(getStackId());
+    }
+
+    @Override
+    protected void executeTracedJob(JobExecutionContext context) throws JobExecutionException {
+        if (flowLogService.isOtherFlowRunning(getStackId())) {
+            LOGGER.debug("StructuredSynchronizerJob cannot run, because flow is running for stack: {}", getStackId());
+            return;
+        }
+        try {
+            StructuredSyncEvent structuredEvent = structuredSyncEventFactory.createStructuredSyncEvent(getStackId());
+            LOGGER.debug(structuredEvent.getStatus());
```

Comment:
Status is not a big help in debugging, it would be better to log stack name, status etc.  Some more words could be helpful for debugging also, like "Collected data to send: {}", structuredEvent